Dockerfile

```diff
@@ -0,0 +1,23 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Use the latest python image from DockerHub (3.x)
+FROM python:latest
+
+# Install postgresql dev headers
+RUN apt-get -y update \
+    && apt-get install -y libpq-dev
+
+# Copy local files into the palantiri directory
+COPY . /palantiri/
+
+# Make /palantiri our working directory
+WORKDIR /palantiri
```

Comment:
I personally would not change the `WORKDIR`. If you changed it to help the Pip installation, I think you should be able to do `pip install /palantiri`.